chore: correct Sbanken typography docs

eliland opened the pull request:
Added documentation for Sbanken typography and fixed a few bugs and inconsistencies for the line-height docs in all themes, and involuntary markup changes caused by line breaks done by prettier.

tujoworker reviewed (approved):
Nice work 🙏 looks good!

... and as we talked about, a e2e test would be cool to get in 👍

snorrekim commented on packages/dnb-design-system-portal/src/docs/quickguide-designer/typography/typographic-rules.mdx:
```diff
@@ -14,6 +15,23 @@ The Eufemia typographic scale is as follows:
+<VisibilityByTheme visible="sbanken">
+## In general
+
+Sbanken has two fonts in its profile; Roboto and Maisson Neue. The latter is used mainly for headlines,
```
I believe this is spelled wrong, "Maison"

eliland replied:
I have misspelled this so many places! Search and replace is my friend.

snorrekim reviewed (commented):
Just a small spelling mistake

eliland commented:
Added cypress tests to catch some previously seen issues (line-heights wrong due to changed line-height-basis, p elements inside headlines, ++), and added a few others. We might want to consider adding tests for CSS that is specific for the ui theme in the future.

As a side note @tujoworker, the themeSwitch and pageNavigation tests always failed for me. Could you check if you see the same issue on your end? Suspect there might be some outdated tests there, but I didn't dig too deep.

tujoworker reviewed (approved):
Nice work!

tujoworker commented:
:tada: This PR is included in version 10.1.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@dnb/eufemia/v/10.1.0)
- [GitHub release](https://github.com/dnbexperience/eufemia/releases/tag/v10.1.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
